projects/plugins/jetpack/package.json

```diff
@@ -66,7 +66,9 @@
 	},
 	"dependencies": {
 		"@automattic/calypso-build": "6.1.0",
-		"@automattic/calypso-color-schemes": "1.0.0",
+		"@automattic/calypso-color-schemes": "2.1.1",
```

Comment:
I don't think so. I updated this because `@automattic/components` depends on having >=2.0. I can't find any usage of the package in Jetpack, and the changes were all renaming or removing variable names so I think the SCSS compiler would be failing if any of them was being used.

So the only change possible would be a change in a color somewhere, but looking around the plugin I have not noticed anything obvious.